### PR TITLE
Added image proc as container

### PR DESCRIPTION
**Pull request by luis-camero**

### CODE REVIEW

File: clearpath_sensors/launch/flir_blackfly.launch.py [hunk 1]
```diff
@@ -65,29 +73,65 @@ def generate_launch_description():
         parameters=[parameters, {'parameter_file': param_mapping_file,}],
         output='screen',
         remappings=[
-            ('flir_blackfly/camera_info', 'color/camera_info'),
+            ('flir_blackfly/camera_info', 'camera_info'),
```
Comment (hilary-luo) [inline clearpath_sensors/launch/flir_blackfly.launch.py:76]: In our API documentation camera_info is indicated to be found at `sensors/camera_#/color/camera_info` so perhaps we need to revisit that documentation.

**Review by hilary-luo — APPROVED** [review]

File: clearpath_sensors/launch/flir_blackfly.launch.py (hunk 1 above)
Comment (tonybaltovski) [reply clearpath_sensors/launch/flir_blackfly.launch.py:76]: @hilary-luo it is namespaced above: 
```
    arg_namespace = DeclareLaunchArgument(
        'namespace',
        default_value='platform/sensors/camera_0')
```

File: clearpath_sensors/launch/flir_blackfly.launch.py [hunk 2]
```diff
@@ -49,14 +51,20 @@ def generate_launch_description():
...
-            FindPackageShare('spinnaker_camera_driver'), 'config',
-             'blackfly_s.yaml'
+            FindPackageShare('spinnaker_camera_driver'),
+            'config',
+            'blackfly_s.yaml'
         ]))
 
     arg_namespace = DeclareLaunchArgument(
         'namespace',
         default_value='platform/sensors/camera_0')
```
Comment (tonybaltovski) [inline clearpath_sensors/launch/flir_blackfly.launch.py:61]: This should be `sensors/camera_0` based on https://docs.clearpathrobotics.com/docs/ros/api/sensors_api

File: clearpath_sensors/launch/flir_blackfly.launch.py (hunk 1 above)
Comment (hilary-luo) [reply clearpath_sensors/launch/flir_blackfly.launch.py:76]: That namespacing doesn't bring in the `color` section, that's all I was referring to.

Comment (tonybaltovski) [reply clearpath_sensors/launch/flir_blackfly.launch.py:76]: Ah, yes!

Comment (luis-camero) [reply clearpath_sensors/launch/flir_blackfly.launch.py:76]: I thought about this too, the issue is that `camera_info` is published by the driver and used by all subsequent post-processing images. So, it makes sense to have it at the top level.

File: clearpath_sensors/launch/flir_blackfly.launch.py (hunk 2 above)
Comment (luis-camero) [reply clearpath_sensors/launch/flir_blackfly.launch.py]: Done

**Pull request merged**